CarService.Api/Services/OrderService.cs

```diff
@@ -63,6 +66,28 @@ public OrderService(UserManager<User> userManager, IUnitOfWorkFactory unitOfWork
             }
         }
 
+        public async Task CreateReviewProposition(string email, ReviewPropositionCreationDto reviewPropositionDto)
+        {
+            var user = await _userManager.FindByEmailAsync(email);
+
+            if (user != null)
+            {
+                using (IUnitOfWork unitOfWork = _unitOfWorkFactory.Create())
+                {
+                    IRepository<ReviewProposition> reviewProposition = unitOfWork.Repository<ReviewProposition>();
```

Comment:
Wow, it looks very strange

Reply:
Could you explain what you mean? Maybe we must change IRepository<ReviewProposition> reviewProposition = unitOfWork.Repository<ReviewProposition>(); to var = unitOfWork.Repository<ReviewProposition>(); ?

Thanks for review